bilten-backend/.env.development

```
@@ -0,0 +1,25 @@
+# Development Environment Configuration
+NODE_ENV=development
+PORT=3001
+FRONTEND_URL=http://localhost:3000
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The FRONTEND_URL key should go before the NODE_ENV key

(UnorderedKey)